test/api.test.ts

```diff
@@ -0,0 +1,38 @@
+import fastify from 'fastify';
+import assert from 'assert';
+
+const app = fastify();
+
+// Test for GET /api
+app.inject({
+  method: 'GET',
+  url: '/api'
+}, (err, response) => {
+  if (err) { console.error(err); assert.fail(err); }
+  assert.strictEqual(response.statusCode, 200);
+  assert.strictEqual(response.headers['content-type'], 'application/json; charset=utf-8');
+});
+
+// Test for GET /api/hello
+app.inject({
+  method: 'GET',
+  url: '/api/hello'
+}, (err, response) => {
+  if (err) { console.error(err); assert.fail(err); }
+  assert.strictEqual(response.statusCode, 200);
+  assert.strictEqual(response.headers['content-type'], 'application/json; charset=utf-8');
+});
+
+// Test for GET /api/ping
+app.inject({
+  method: 'GET',
+  url: '/api/ping'
+}, (err, response) => {
+  if (err) { console.error(err); assert.fail(err); }
+  assert.strictEqual(response.statusCode, 200);
+  assert.strictEqual(response.headers['content-type'], 'application/json; charset=utf-8');
+});
+
+app.close().catch((err) => console.error(err));
```

Comment:
Please add `@typescript-eslint/no-confusing-void-expression` on top of file to fix the linter issue.

Reply:
Not yet.